internal/relic/cavern/hunterofglacialforest/hunterofglacialforest.go

```diff
@@ -0,0 +1,63 @@
+package hunterofglacialforest
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/relic"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	mod  = key.Modifier("hunter-of-glacial-forest")
+	buff = key.Modifier("hunter-of-glacial-forest-buff")
+)
+
+// 2pc: Increase Ice DMG by 10%
+// 4pc: After the wearer uses their Ultimate, their CRIT DMG increases by 25% for 2 turn(s)
+
+func init() {
+	relic.Register(key.HunterOfGlacialForest, relic.Config{
+		Effects: []relic.SetEffect{
+			{
+				MinCount: 2,
+				Stats:    info.PropMap{prop.IceDamagePercent: 0.10},
+			},
+			{
+				MinCount: 4,
+				CreateEffect: func(engine engine.Engine, owner key.TargetID) {
+					engine.AddModifier(owner, info.Modifier{
+						Name:   mod,
+						Source: owner,
+					})
+				},
+			},
+		},
+	})
+
+	modifier.Register(mod, modifier.Config{
+		Listeners: modifier.Listeners{
+			OnBeforeAction: onBeforeAction,
+		},
+	})
+
+	modifier.Register(mod, modifier.Config{
+		Stacking:   modifier.ReplaceBySource,
+		StatusType: model.StatusType_STATUS_BUFF,
+	})
```

Comment:
Need to register under the `buff` modifier key (tests currently failing because you are registering 2 configs to the same modifier name)

Reply:
fixed!